plat/qemu/qemu_image_load.c

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright (c) 2017, Linaro Limited. All rights reserved.
```

Comment:
@davidcunado-arm 
Since I am a Linaro assignee, So I just check the existing file for the Copyright, I found a services/spd/opteed/teesmc_opteed.h  has this. So I copy and paste it.
So I should use 
Copyright (c) 2017, ARM Limited and Contributors. All rights reserved.
Right??